Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.h

```diff
@@ -225,12 +225,18 @@ class TetrahedronFEMForceField : public core::behavior::ForceField<DataTypes>, p
     Data<type::vector<Real> > _vonMisesPerElement; ///< von Mises Stress per element
     Data<type::vector<Real> > _vonMisesPerNode; ///< von Mises Stress per node
     Data<type::vector<type::Vec4f> > _vonMisesStressColors; ///< Vector of colors describing the VonMises stress
-    
+
+    Real _minVMN;
+    Real _maxVMN;
```

Comment:
```suggestion
    Real m_minVonMisesPerNode;
    Real m_maxVonMisesPerNode;
```

1. According to the guidelines, private class members must start with `m_`
2. The name `minVMN` is not clear